README.md

```diff
@@ -1,44 +1,251 @@
+
+- During set-up, you need to create a GitHub personal access token and a CircleCi access token; you should store your generated tokens securely.
+
+- For your CI build to work, you need to opt into the use of third-party orbs you can find this in your CircleCi Org Setting > Security > Allow Uncertified Orbs.
+
+- It takes about 20 - 35 mins for your deployed frontend application to be globally available through AWS CloudFront CDN.
+
 
-#### Building this tool
+### zero init
+
+The `zero init` command creates a new project and outputs an infrastructure configuration file with user input prompted responses into a file.  -> 📁 `YOUR_PROJECT_NAME/zero-project.yml`
+
+```shell
+# To create and customize a new project you run
+$ zero init
+
+## Sample project initilization
+✔ Project Name: zero-test
+🎉  Initializing project
+✔ EKS + Go + React
+✔ Should the created projects be checked into github automatically? (y/n): y
+✔ What's the root of the github org to create repositories in?: github.com/zero-test-org
+✔ Existing AWS Profiles 
+✔ default
+
+Github personal access token: used for creating repositories for your project
+Requires the following permissions: [repo::public_repo, admin::orgread:org]
+The token can be created at https://github.com/settings/tokens
+✔ Github Personal Access Token with access to the above organization: <MY_GITHUB_ORG_ACCESS_TOKEN>
+
+CircleCI api token: used for setting up CI/CD for your project
+The token can be created at https://app.circleci.com/settings/user/tokens
+✔ Circleci api key for CI/CD: <MY_CIRCLE_CI_ACCESS_TOKEN>
+✔ us-west-2
+✔ Production Root Host Name (e.g. mydomain.com) - this must be the root of the chosen domain, not a subdomain.: commitzero.com
+✔ Production Frontend Host Name (e.g. app.): app-zero-test-prod.
+✔ Production Backend Host Name (e.g. api.): api-zero-test-prod.
+✔ Staging Root Host Name (e.g. mydomain-staging.com) - this must be the root of the chosen domain, not a subdomain.: commitzero-stage.com
+✔ Staging Frontend Host Name (e.g. app.): app-zero-test-stage.
+✔ Staging Backend Host Name (e.g. api.): api-zero-test-stage.
+✔ What do you want to call the zero-aws-eks-stack project?: infrastructure
+✔ What do you want to call the zero-deployable-backend project?: backend-service
+✔ What do you want to call the zero-deployable-react-frontend project?: frontend
+
+```
+
+### zero create
+
+The `zero create` command renders the infrastructure modules you've configured into your project folder and pushed your code to GitHub.
+
+```shell
+# Template the selected modules and configuration specified in zero-project.yml and push to repository.
+$ cd zero-init   # change your working dir to YOUR_PROJECT_NAME
```

Comment:
maybe change the readme project name to something more familiar like [helloworld/myapp/my-k8s-stack]
and have the project name consistent through the commands so its not `zero-project` during `init` and `zero-test` here